Energies.jl

```diff
@@ -0,0 +1,10 @@
+G_values = [-5.92, -5.942, -5.97, -5.948, -5.937, -5.94] #Amorphous unsure
+deltaG = [G_values[j] - G_values[i] for j in 1:length(G_values), i in 1:length(G_values)]
+Ea_constants = [0.00 1.00 0.01 0.01 1.00 1.00;  #set the activation energy from amorphous to kappa and beta to be small

```

Comment:
if you declare this as a `Symmetric` matrix, you'll save a little bit of memory and also guard against typos etc.